### ISSUE TITLE
atis数据集模型参数应如何设置

### ISSUE THREAD

**Issue by Jutao1208** [czhang99/Capsule-NLU#1]
我这边论文复现，snips数据集按照论文的参数设置，效果能达到预期效果，但是atis数据集的Overall Acc只能达到0.704，与之对应的Intent Acc只有0.776，并且训练过程intent loss 一直震荡不收敛，已经尝试了不同的学习率和优化器，效果均达不到论文水平，请问一下作者论文关于atis数据集的超参是怎么设置的，在此十分感谢！！

**Comment by benjamingli** [comment]
是的，我也有同样的问题，望解答

**Comment by Jutao1208** [comment]
将intent的训练参数改为
intent_params = tf.trainable_variables()
因为slot层的输出到intent层

**Comment by benjamingli** [comment]
有效！谢谢！
